src/nvim/os/shell.c

```diff
@@ -162,6 +163,8 @@ int os_call_shell(char_u *cmd, ShellOpts opts, char_u *extra_args)
 /// @param input The input to the shell (NULL for no input), passed to the
```

Comment:
The comments on `argv`, `input`, and `output` are also outdated (@splinterofchaos please confirm): `do_os_system()` does not internally make any decision about sending a command to the shell--it invokes `argv` directly, which may or may not be a command like `bash -c ...`.

Instead of `shell`, just `process` seems correct.